integration-tests/tests/cli/__snapshot_serializers__/cli-output.ts

```diff
@@ -0,0 +1,73 @@
+import stripAnsi from 'strip-ansi';
+import type { SnapshotSerializer } from 'vitest';
+import { ExecaError } from '@esm2cjs/execa';
+
+export const cliOutput: SnapshotSerializer = {
+  test: (value: unknown) => {
+    if (typeof value === 'string') {
+      return variableReplacements.some(replacement => replacement.pattern.test(value));
+    }
+    return isExecaError(value);
+  },
+  serialize: (value: unknown) => {
+    if (typeof value === 'string') {
+      let valueSerialized = '';
+      valueSerialized += ':::::::::::::::: CLI SUCCESS OUTPUT :::::::::::::::::\n';
```

Comment:
I really like the format you went with for cli output. It captures all the important data and is very readable.
My only recommendation again would be that this be in a separate PR to isolate the json output work.

Reply:
Do you have a proposal about how to test the changes in this PR? Removing this would reduce confidence in the PR changes would it not?

Reply:
Internal team discussion led to a decision to make other smaller PRs. Let's see where that goes. May affect this thread.